numpy/linalg/umath_linalg.c.src

```diff
@@ -706,6 +706,11 @@ dump_@TYPE@_matrix(const char* name,
  *****************************************************************************
  */
 
+#ifndef min
+#define min(x,y) ((x<y)?x:y)
```

Comment:
Probably. There are lots of small things like that scattered through existing code that need fixing but don't have a high priority. For new code we try to get it right...